guides/common/modules/snip_supported-client-operating-systems.adoc

```diff
@@ -7,6 +7,11 @@ ifndef::orcharhino[]
 endif::[]
 ifdef::satellite[]
 * {EL} 7 and 6 with the https://www.redhat.com/en/resources/els-datasheet[ELS Add-On]
+ifdef::managing-hosts[]
+* You can register the following hosts for xref:converting-a-host-to-rhel_{context}[converting to RHEL]:
```

Comment:
I recommend removing this xref for 2 reasons:
- The user can easily find the topic in the ToC, so it is not necessary.
- Adding an xref to a snippet, which is intended to be used frequently, introduces a vulnerability.